.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,207 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches: []
+    eventResponderTasks:
+    - if:
+      - payloadType: Pull_Request
+      - hasLabel:
+          label: auto-merge
+      then:
+      - enableAutoMerge:
+          mergeMethod: Merge
+      description: Automatically merge PR once CI passes
+    - if:
+      - payloadType: Pull_Request
+      - labelRemoved:
+          label: auto-merge
+      then:
+      - disableAutoMerge
+      description: Automatically merge PR once CI passes
+    - if:
+      - payloadType: Pull_Request
+      - labelAdded:
+          label: auto-merge
+      - isActivitySender:
+          user: dotnet-bot
+          issueAuthor: False
+      then:
+      - approvePullRequest:
+          comment: Auto-approval of dotnet-bot PRs
+      description: Auto-approve PRs labeled with "auto-merge" opened by dotnet-bot
+    - if:
+      - payloadType: Pull_Request
+      - isAction:
+          action: Null

```

Comment:
All of the rules that were checking for merged PRs are broken here. I'll push a commit to fix them.